src/main/java/com/server/nodak/exception/common/BaseException.java

```diff
@@ -0,0 +1,14 @@
+package com.server.nodak.exception.common;
+
+import lombok.Getter;
+
+@Getter
+public class BaseException extends RuntimeException {
+
+    int code;
```

Comment:
```suggestion
    private int code;
```
Getter 가 있어서 위 Private 제어자를 붙이는 것은 어떨까요?

Reply:
넵 좋습니다!